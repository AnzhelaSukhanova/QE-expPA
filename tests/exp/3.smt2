(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (or (bvult (bvadd (bvshl #b0001 x) x) (bvadd (bvadd (bvshl #b0001 y) y) #b0001)) (= (bvadd (bvshl #b0001 x) x) (bvadd (bvadd (bvshl #b0001 y) y) #b0001)))))
