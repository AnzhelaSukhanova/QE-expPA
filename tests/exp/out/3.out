(let (($e1 (bvadd #x1 (bvadd y (bvshl #x1 y))))) (let (($e2 (bvult #x6 $e1))) (not (and (not (bvult #x3 $e1)) (not (bvult #x1 $e1)) (not $e2) (not (bvult #xb $e1)) (not (bvult #x4 $e1)) (not (bvult #x5 $e1)) (not $e2)))))
